src/controllers/dashboard/dashboard.html

```diff
@@ -44,7 +44,7 @@ <h3>${HeaderActiveDevices}</h3>
 
             <div class="dashboardColumn dashboardColumn-2-40 dashboardColumn-3-27">
                 <div class="dashboardSection">
-                    <a is="emby-linkbutton" href="#/serveractivity.html?useractivity=true" class="button-flat sectionTitleTextButton">
+                    <a is="emby-linkbutton" href="#/dashboard/activity?useractivity=true" class="button-flat sectionTitleTextButton">
```

Comment:
Does this change the non-experimental route to go to the experimental route from the default dashboard?

Reply:
Nope the same route is used for both. There are separate sets of route definitions for the experimental and stable layouts so we can have the same route render the appropriate view depending on which app is active.

Reply:
:magic_wand: